### DISCUSSION TITLE
夸克网盘无法播放视频

### DISCUSSION THREAD

**Discussion by Nicobalice** [AlistGo/alist#4143] [General]
本人用的raidrive挂载本地。白天还能正常使用，刚刚突然不能在线播放，去alist点开视频，5次重新连接之后显示加载失败，点击下载显示{"code":500,"message":"failed link: failed get link: download file size limit[4d4dc0d969ac42259beb49bea46ac1b0]","data":null}除了视频其他都能正常看。同时还挂载了一个阿里网盘，这个就一切正常。求大佬解答。谢谢🙏

**Comment by Nicobalice** [comment]
重新挂载 更改webdav策略 更新cookie 重启 都尝试过但无法解决

**Comment by fhanfc** [comment]
一样  我也是下午好好的看着动漫  傍晚就看不成了  本以为是自己下载量过大夸克给我限流了  但是打开别的视频发现 一些特别小的视频可以打开   同时在夸克网页端   本可以通过油猴插件下载的大文件  全都变得无法获取链接 我估计是夸克吧网页端的链接给改掉了  导致alist获取不到大资源的链接   所以没法看视频   自己想的  不一定对  我估计夸克是寄了

**Reply by Nicobalice** [reply]
555 难受啊 我以为就我遇到了 谢谢

**Comment by Nicobalice** [comment]
我刚刚看贴吧也有人发了 应该今天晚上出的问题 大家都寄了

**Comment by KinShin** [comment]
油猴脚本网盘直链下载助手也无法下载大文件，应该是夸克网盘官方做的限制。

**Reply by Nicobalice** [reply]
谢谢解答 那后续会解决吗

**Comment by Nicobalice** [comment]
今天更新3.16版本后可以打开了 不过速度好慢啊

**Reply by Unicor023** [reply]
我这现在还是不行啊，网页也放不了

**Reply by Nicobalice** [reply]
我的最快不到200KB/S，虽然不至于不停重连，但也属于半死不活，还是无法使用。

**Comment by juanlian687** [comment]
我也遇到了，有解决办法吗?

**Reply by Nicobalice** [reply]
暂时还没有

**Comment by DarinJi** [comment]
一样，如何解决?

**Reply by Nicobalice** [reply]
暂时还没有

**Comment by mario8921** [comment]
infuse+alist（挂载夸克盘）+Synology NAS，昨晚想看电影，结果就视频出现无法加载的情况，还以为是网络问题，原来都有这个问题啊，使用网页播放，连接5s后就显示加载失败，重新更改了cookie还是不行，求大佬解救

**Reply by Nicobalice** [reply]
暂时只能更最新版本 虽然不会加载失败了，但最快也就200kb
